packages/pdf-renderer/src/lib/reactRenderer.service.ts

```diff
@@ -30,4 +30,46 @@ export class ReactRenderer {
             </html>
         `;
     }
+
+    generateWithHeaderAndFooter(
```

Comment:
Do we need another method, if both header and footer are optional already?

Reply:
Yes, this method should emulate the behavior of including header and footer using `Page.pdf()` from puppeteer. This is mostly for debugging purposes because previously it was impossible to inspect elements and styling of header and footer elements when chosing html as output format. Even with current implementation of this method there are some changes that should be separate from `generate` method e.g. `@page` styling which is used here only for emulating the layout of a generated pdf if somone wants to see the print preview of html output. Of course this could have been done with if statements but it is quite likely that more changes will be made specifically to emulate the behavior of including header and footer with `Page.pdf()` and including them in one method with conditional rendering just doesn't feel right